ui/app/src/modules/Content/Dependencies/DependencySelection.tsx

```diff
@@ -291,7 +291,7 @@ export function DependencySelectionDelete(props: DependencySelectionDeleteProps)
       .filter(([key, value]) => value === true)
       .map(([key]) => key);
     onChange(result);
-  }, [checked, onChange]);
+  }, [checked]);
```

Comment:
Removed dependency. If dev creates loop, useCallback where needed

Reply:
Jose will fix this issue